snippets/rust.json

```diff
@@ -416,6 +416,13 @@
         ],
         "description": "impl … { … }"
     },
+    "inline": {
+        "prefix": "inline",
+        "body": [
+            "#[inline]"
```

Comment:
You have two choices:

### First 

`git reset --hard HEAD~1` - it will remove your commit from history
Make your changes
`git commit`
`git push --force` - it will replace the current commit with a new one

### Second

`git revert HEAD` - it will revert your commit.
Make your changes
`git commit`
`git push`